packages/wdio-browserstack-service/src/constants.ts

```diff
@@ -70,6 +70,15 @@ export const LOG_KIND_USAGE_MAP = {
 
 export const FUNNEL_INSTRUMENTATION_URL = 'https://api.browserstack.com/sdk/v1/event'
 
+export const SUPPORTED_BROWSERS_FOR_AI = ['chrome', 'microsoftedge', 'firefox']
+
+export const TCG_URL = 'https://tcg.browserstack.com'
```

Comment:
Any reason why we haven't integrated with TestHub?

Reply:
Not sure about TestHub Integration. This flow is separate